Assets/MRTK/Core/Providers/InputSimulation/SimulatedMotionController.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Snapshot of simulated motion controller data.
+    /// </summary>
+    public class SimulatedMotionControllerData
+    {
+        [SerializeField]
+        private bool isTracked = false;
+        public bool IsTracked => isTracked;
+        [SerializeField]
```

Comment:
nit: missing doc comments for the public properties here and below
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#function--method-summary-headers

Reply:
Comments added.